content/android/_index.md

```diff
@@ -11,4 +11,8 @@ listdescription: ""
 
```

Comment:
@pdavidc I'm on board with combining the two pages. The question then becomes what should the "default" page for the projects become, and how does its URL look, and how should the project navigation look?

For example, when you go to to `localhost:1313/android/` right now, you get the overview page, which is the "default" page and represented by the "Overview" link in project navigation.

For clarification, "default" has no specific Hugo connotation, I'm speaking in more general permalink/url and site behavior.

I think there a few options. I've outlined them below. Each one addresses the default page, URL pattern, and project navigation changes. I'm leaning towards the first one.

1. "Get Started" becomes the default. The `/android/get-started/` url disappears. Project navigation removes the current "Overview" option and starts with the "Get Started" option. For completeness, we can use Hugo [Aliases](https://gohugo.io/extras/aliases/) to redirect `/android/get-started/` to the default `/android/`.
2. "Get Started" is renamed to something more generic (i.e. Overview, Home, etc.) and the `/android/get-started` URL disappears. The menu option for "Get Started" is replaced with the more generic selection.
3. We come up with more content for the current Overview or implement some other "default" or introduction page.

Actually, as I've been documenting this, I really think the first one is the best option. I'm open to any suggestions or comments, but I'm going to implement the first option until hearing otherwise.